[quant] Un-hardcode epsilon constant in FoldConvBatchNorm2d.

ZolotukhinM opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#26584 [quant] Un-hardcode epsilon constant in FoldConvBatchNorm2d.**
* #26581 Register values listed in __constants__ as attributes of the Module.

Differential Revision: [D17514653](https://our.internmc.facebook.com/intern/diff/D17514653)

Commit:
[quant] Un-hardcode epsilon constant in FoldConvBatchNorm2d.

[ghstack-poisoned]

jerryzh168 commented on test/test_jit.py:
```diff
@@ -1258,15 +1258,15 @@ class TestModule(torch.nn.Module):
 
         eager = TestModule()
-        scripted = torch.jit.script(eager)
+        scripted = torch.jit.script(copy.deepcopy(eager))
```
why do we need to copy the eager module here?

ZolotukhinM replied:
Otherwise the scripted and eager model share the same tensors. It was a bug previously and I think it worked by luck.

jerryzh168 reviewed (approved):
LGTM, had a question, see inline comment

Commit:
Add tests to verify that we can't mutate constant attributes. on "[quant] Un-hardcode epsilon constant in FoldConvBatchNorm2d."


Differential Revision: [D17514653](https://our.internmc.facebook.com/intern/diff/D17514653)

[ghstack-poisoned]

Commit:
Update on "[quant] Un-hardcode epsilon constant in FoldConvBatchNorm2d."


Differential Revision: [D17514653](https://our.internmc.facebook.com/intern/diff/D17514653)

[ghstack-poisoned]
